crates/moonutil/src/scan.rs

```diff
@@ -391,6 +415,24 @@ fn scan_one_package(
             )
         }))
     };
+
+    let mut prebuild = pkg.pre_build.unwrap_or(vec![]);
+    for mbt_x_file in mbt_x_files {
+        // mbt_x_file is a file with extension .mbt.x
+        // mbt_file is the same file with extension .mbt
+        let mbt_file = mbt_x_file.with_extension("");
+        let generate = MoonPkgGenerate {
+            input: crate::package::StringOrArray::String(mbt_x_file.display().to_string()),
```

Comment:
The `StringOrArray` conflicts with `cond_expr::StringOrArray`. Maybe we can make `StringOrArray` a shared type?